### DISCUSSION TITLE
Keep custom theme during navigation on Daisyui site

### DISCUSSION THREAD

**Discussion by Suniron** [saadeghi/daisyui#1875] [Ideas / feature requests]
Hey,

Is it possible to keep the customized daisyui theme during the navigation on the official website?

I think it's a good idea to have a real preview of components

**Comment by saadeghi** [comment]
It was like that at first but turned out it wasn't a good idea. Theme generator allows people to customize all colors and they are also allowed to change colors to some really bad contrast after that, it will destroy the whole site experience unless the y go back to theme generator and fix it.

That's why I decided to put a compact box to show the important components instead and only that box is affected by the theme generator.
For more tests, you can put the new theme in your site and see your own project. (it's just a color picker after all and you can edit the colors on your code editor as well)

Also you can put your theme in Tailwind Play and play with it there: https://daisyui.com/tailwindplay/

Let me know if you have any questions.

**Reply by Suniron** [reply]
Thanks for you explanation, I understand!

If all components are up to date on the ''Theme generator page'', it seems to be OK!
